src/lib-imap/Makefile.am

```diff
@@ -96,3 +96,16 @@ check-local:
 	for bin in $(test_programs); do \
 	  if ! $(RUN_TEST) ./$$bin; then exit 1; fi; \
 	done
+
+if USE_FUZZER
+noinst_PROGRAMS += \
+	fuzz-imap-utf7
+
+nodist_EXTRA_fuzz_imap_utf7_SOURCES = force-cxx-linking.cxx
```

Comment:
is this file generated or what? why not put it in EXTRA_SOURCES outside the if block?

Reply:
It's a kludgy way to use C++ linking. The file doesn't actually exist. It's recommended in https://www.gnu.org/software/automake/manual/html_node/How-the-Linker-is-Chosen.html

Reply:
Although it would be nicer if this was done across all binaries so it wouldn't have to be repeated for every single fuzzer binary.. Since this is only used for fuzzing, it wouldn't matter if other binaries were also linked with clang++. I'm not sure if there's any way to do that though.

Reply:
I suppose the fuzz*.c sources could be C++ sources directly, but I'm not sure if that causes problems. And could be nice to be able to compile the fuzzers also when C++ wasn't required to be used.